test/integration/connect/envoy/case-dogstatsd-udp/s1.hcl

```diff
@@ -1,25 +1,30 @@
 services {
   name = "s1"
   port = 8080
+
   connect {
     sidecar_service {
+      port = 21000
```

Comment:
Was this added to _all_ the tests? I thought the goal was to test explicit setting as well as the default auto-assign, did I misunderstand?

Reply:
I wanted to ensure that the proxy ports all came up deterministically when there were a lot of services being loaded. Though on second thought and a skim through the way service HCL files are bootstrapped it should be sufficiently deterministic by the asciibetically sorted filenames.

I think I may have done the explicit port thing when chasing that snapshot/restore agent bug. I'll see if everything is still ok without the explicit wiring.

Reply:
I found why i thought i needed it:
`s1, s2, s2-v1, s2-v2` `sort.Strings` as `s1, s2, s2-v1, s2-v2` but consul's config loader loads them as `s1, s2-v1, s2-v2, s2`.

This is because I forgot the extension plays a role:
`s1.hcl, s2.hcl, s2-v1.hcl, s2-v2.hcl` `sort.Strings` as `s1.hcl, s2-v1.hcl, s2-v2.hcl, s2.hcl`